index.js

```diff
@@ -1,17 +1,46 @@
 import glamorous from 'glamorous'
 
+// A <div>
 const View = glamorous.div()
+
+// Blocks
 const Block = glamorous.div({display: 'block'})
 const InlineBlock = glamorous.div({display: 'inline-block'})
+const InlineListItem = glamorous.div({display: 'inline-list-item'})
+
+// Flex
 const Flex = glamorous.div({display: 'flex'})
 const Row = glamorous.div({display: 'flex', flexDirection: 'row'})
 const Column = glamorous.div({display: 'flex', flexDirection: 'column'})
 
+// Tables
+const Table = glamorous.div({display: 'table'})
```

Comment:
You could make all this more terse by adding `export` to every const declaration.

```js
export const Table = ...
```

Then you could drop the export at the bottom